solutions/baseline/version.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  # module uses nullable feature which is only available in versions >= 1.1.0
+  required_version = ">= 1.3.0"
```

Comment:
Can you bump this to >= 1.9.0 and start to use cross variable validation for a better consumer experience of the DA? Because we support disabling cloud logs and/or monitoring agent we badly need validation to ensure the required inputs are caught by validation rules